src/components/OnsAnalyticsBanner.svelte

```diff
@@ -5,7 +5,8 @@
   export let analyticsId; // Required. Google analytics/tag manager ID
   export let analyticsProps = {}; // Optional props to describe the content
   export let usageCookies = false; // True if usage cookies are allowed (to be read from parent component)
-  export let fixed = true; // Fixed positioning of banner (instead of inline)
+  // ToDo is the fixed property needed at all?
+  // export let fixed = true; // Fixed positioning of banner (instead of inline)
 
```

Comment:
Not needed, so let's delete - I guess it won't work without the style you've deleted anyways

Reply:
same for all others?